cosmos/carbon.json

```diff
@@ -153,9 +153,9 @@
       "coinDecimals": 18,
       "coinGeckoId": "ethereum",
       "gasPriceStep": {
-        "low": 76923076.92307693,
-        "average": 96153846.15384616,
-        "high": 115384615.38461539
+        "low": 33333333.333333333333,
+        "average": 33333333.333333333333,
+        "high": 33333333.333333333333
```

Comment:
Same issue as comment on L104, need to round ceil, if not user will be paying insufficient gas. Also, follow the original number, round to 8 d.p.